math/sqrt_double.cpp

```diff
@@ -6,6 +6,9 @@ number in O(logn) time,
 with precision fixed */
 
 double Sqrt(double x) {
+    if ( x > 0 && x < 1 ) {
```

Comment:
This is the only part that is new. 

The pull-request is invalid. Please see some tutorials on how to commit and create a pull-request - [example link](https://opensource.com/article/19/7/create-pull-request-github) 

If you are contributing, contribute a new file and not change existing one unless you are making any corrections to them. 

Thank you